tests/functional/testplan/test_timeout.py

```diff
@@ -62,3 +62,4 @@ def test_runner_timeout():
         assert current_proc.children() == start_procs
     finally:
         os.remove(output_json)
+        pass
```

Comment:
Why is this needed...?

Reply:
probably added for debugging purpose, removed.